models/filter/utils.go

```diff
@@ -15,3 +17,9 @@ func applyConditionsIfNotNil(conditions map[string]interface{}, columnName strin
 func ptr[T any](value T) *T {
 	return &value
 }
+
+func fromJSON[T any](raw string) T {
+	var filter T
+	_ = json.Unmarshal([]byte(raw), &filter)
```

Comment:
What if there is an error?

Reply:
The `fromJSON` private function is used only in tests.

Reply:
Will move it to `utils_test.go` file

Reply:
Done